<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveTreatment</class>
 <widget class="QDialog" name="ActiveTreatment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLCDNumber" name="curr_Freq">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QOpenGLWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>300</width>
     <height>230</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>20</width>
     <height>291</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Time of Therapy : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="skinTouch">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Touch Here</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="therapyTime">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>64</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>322</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="freq_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
